WordPress/src/main/java/org/wordpress/android/util/WPWebViewClient.java

```diff
@@ -45,6 +42,12 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
         if (!url.equals("file:///android_asset/webkit/")) {
```

Comment:
This check needs to be changed, otherwise `view.loadUrl` may be called twice (once when the URL isn't `file:///android_asset/webkit/` and again when the blog is private).